lib/winston-mongodb.d.ts

```diff
@@ -3,13 +3,13 @@
 // Definitions by: miton18 <https://github.com/miton18>, blove <https://github.com/blove>,
 // Balazs Mocsai <https://github.com/mbale>
 
-import { TransportInstance, Transports, TransportOptions } from "winston";
+import { Transports } from "winston";
```

Comment:
```suggestion
import { transports } from "winston";
```

Reply:
This should be in lower case as it is how is exported from winston

Reply:
@uxweb : Your suggested changes works fine. Thanks!